config/config.ts

```diff
@@ -1,22 +1,60 @@
-const network = '250';
+// const network = '250';
```

Comment:
Just for the future - commenting a lot of consecutive lines this way erases blame history and creates unnecessary diffs, better option is to use /* */ on lines above and below the code you want to comment out